api/openlattice.yaml

```diff
@@ -2503,6 +2505,45 @@ paths:
                 items:
                   type: string
                   format: uuid
+  /datastore/data/set/{entitySetId}/filtered:
+    post:
+      summary: Loads data in multiple pages
+      operationId: loadFilteredEntitySetData
+      parameters:
+        - name: entitySetId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
```

Comment:
Just want to double check that "array" is expected here -- the body should just be a single FilteredDataPageDefinition object

Reply:
Also think this should be object.